### DISCUSSION TITLE
Project 4

### DISCUSSION THREAD

**Discussion by ElisDiaz46** [cooperrc/computational-mechanics#182] [Q&A]
Hello,

I hope all is well. I don't necessarily how to answers these questions. Does these required further coding? I already did part 1 and part 2. 

![image](https://user-images.githubusercontent.com/116235922/212477512-8de14747-89ac-4d42-8771-7eed11bd06a2.png)

**Comment by DJA17** [comment]
You can just edit the code you used to find ufree and afjust the F to be F[5] = -300/70e3/0.1/46.07142857*0.2 where you divide the area by the highest number in the array -46.07*0.2 where 0.2 is the desired area. You can di this for a and b. For c, you can look up the densities and multiply by the number of nodes and the cross sectional area. Hope this helps
